Policy: add "rates" (fees + subsidies) (#484)
openmobilityfoundation/mobility-data-specification: policy/README.md

| `count` | Fleet counts based on regions. Rule `max`/`min` refers to number of devices.                                  |
| `time`  | Individual limitations on time spent in one or more vehicle-states. Rule `max`/`min` refers to increments of time in [Rule Units](#rule-units). |
| `speed` | Global or local speed limits. Rule `max`/`min` refers to speed in [Rule Units](#rule-units).                  |

> | `rate`  | **[Beta feature](/general-information.md#beta-features):** Yes (as of 1.0.0). Fees or subsidies based on regions and time spent in one or more vehicle-states. Rule `rate_amount` refers to the rate charged according to the [Rate Recurrence](#rate_recurrence). Agencies and Providers must agree on terms of use prior to utilizing the `rate` rule type.     |

Reviewer: ```suggestion
| `rate`  | **[Beta feature](/general-information.md#beta-features):** Yes (as of 1.0.0). Fees or subsidies based on regions and time spent in one or more vehicle-states. Rule `rate_amount` refers to the rate charged according to the [Rate Recurrence](#rate_recurrence). As this is a beta feature, agencies must communicate with providers about how the `rate` rule type will be used prior to implementation.     |
```

Reviewer 2: I prefer the earlier language, but maybe agree gives providers more say than cities are willing to give. If "communicate" was changed to "consult" that makes it feel more like the needs of providers should be considered when implementing.

Reviewer: "Consult" sounds fine to me. We'll highlight this language for the Board too when they do their review of the release.
